common/lib/protocol-definitions/package.json

```diff
@@ -11,14 +11,40 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
-	"main": "dist/index.js",
-	"module": "lib/index.js",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/index.d.mts",
+				"default": "./lib/index.mjs"
+			},
+			"require": {
+				"types": "./dist/index.d.ts",
+				"default": "./dist/index.cjs"
+			}
+		},
+		"./alpha": {
```

Comment:
It sounds like the plan of record (a somewhat new development) is that we want to make our import alias for our `@alpha` rollups be `/fruit` (both to ease Loop's concerns and to make a bit more explicit the fact that we are not really using `@alpha` for its intended purpose).

Shouldn't be a problem to wait until later to rename these as needed, just figured bring it to your attention.

Reply:
I just removed it for now.